metron-interface/metron-rest/src/main/scripts/metron-rest.sh

```diff
@@ -47,6 +55,15 @@ rest_jar_pattern="${METRON_HOME}/lib/metron-rest*.jar"
 rest_files=( ${rest_jar_pattern} )
 echo "Default metron-rest jar is: ${rest_files[0]}"
 METRON_REST_CLASSPATH+=":${rest_files[0]}"
+METRON_REST_CLASSPATH+=":$PARSER_LIB"
+
+if [ -d "$PARSER_CONTRIB" ]; then
+  contrib_jar_pattern="${PARSER_CONTRIB}/*.jar"
+  contrib_list=( $contrib_jar_pattern ) # expand the glob to a list
```

Comment:
This is probably ok per our supported OS's and shells, but I don't think this is cross platform. It does not work in ZSH, for instance.

Reply:
It's definitely not cross-shell.  The shell scripts we have currently all shbang to bash on the first line for that reason.